src/runtime/loader.ts

```diff
@@ -160,7 +160,15 @@ export class Loader {
    */
   unwrapParticle(particleWrapper): typeof Particle {
     assert(this.pec);
-    return particleWrapper({Particle, DomParticle, TransformationDomParticle, MultiplexerDomParticle, Reference: ClientReference.newClientReference(this.pec), html});
+    return particleWrapper({
+      Particle,
+      DomParticle,
+      SimpleParticle: DomParticle,
```

Comment:
so this is still using DomParticle, right?

Reply:
Only the test infrastructure uses the `runtime/loader.ts` as is, and my diagnosis (which could be wrong) is as above: we didn't build the tests to be careful about infrastructure under test vs infrastructure used as harness.